http/httpclient/src/main/java/com/okta/commons/http/httpclient/HttpClientRequestExecutor.java

```diff
@@ -332,21 +333,24 @@ private void parseRequestExecutorParams(Map<String, String> props) {
     }
 
     private Integer lookupConfigValue(Map<String, String> props, String key, String sysPropName) {
+        // check pops first
+        String configuredValue = props.get(key);
```

Comment:
This code will work incorrectly when the `configuredValue` property has not empty not valid to be parsed as an Integer string.

Reply:
@sergiishamrai-okta I'm not sure I'm following

`props.get(key)` will return a string or null, if null the sys prop is looked up, if/when that is non-null the string is parsed to an int (otherwise `null` is returned and the default value is used above)

Reply:
The logic is changed here.
Code in master will try to read and parse the value and if it fails then go to the next "source" of config. In the branch, the code will get any value non-null-non-empty and try to parse it at the very end. So the ability to read the next correct value is lost.
It's not an issue if we can assume that the config could contain only correct values. The first incorrect value in config will fail the application start even if the lower-level config contains the correct value.

Reply:
I see what you mean now, I didn't realize those tests were skipped, so I assumed my code was valid 🤦 
I fixed those tests so they are no longer skipped (and the corresponding logic)

Reply:
The code seemed fine to me earlier and I overlooked the test failure. Thanks for fixing it @bdemers!